src/rollup/pathfora.js

```diff
@@ -0,0 +1,104 @@
+/** @module pathfora */
+
+import { PF_VERSION, PF_LOCALE, PF_DATE_OPTIONS, CSS_URL } from './globals/config';
+
```

Comment:
might be worth adding some comments to each section so its clear whats loaded and why. will be helpful when adding new modules in order to keep similar functionality grouped.